docusaurus/docs/Android/02-client/06-guides/01-push-notifications/02-setup.mdx

```diff
@@ -0,0 +1,280 @@
+
+You can provide custom icon and text resources displayed in notifications by overriding them:
+- We are using [stream_ic_notification.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-client/src/main/res/drawable/stream_ic_notification.xml) drawable as the default notification's icon
+- Text resources are placed inside [string.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-client/src/main/res/values/strings.xml) file
+
+:::info
+The SDK supports multiple languages. Make sure to override resources in all languages used in your app. Refer to [Adding Localization](../../../03-ui/06-guides/04-custom-translations.mdx) guide for more details.
+:::
+
+### Using our `NotificationHandlerFactory`
+
+Our `NotificationHandlerFactory` provides a way to create a default implementation of `NotificationHandler` based on the Android API Level and lets you customize an intent that a user triggers by clicking the notification.
+This function lets you provide a custom `Intent` that can point to any `Activity` in your app. You can also add custom data to the `Intent` if you need it.
+
+<Tabs>
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin {1-12,15}
+val notificationHandler = NotificationHandlerFactory.createNotificationHandler(
+    context = context,
+    newMessageIntent = {
+        messageId: String,
+        channelType: String,
+        channelId: String,
+         ->
+        // Return the intent you want to be triggered when the notification is clicked
+        val intent: Intent = [....]
+        intent
+    }
+)
+
+ChatClient.Builder("api-key", context)
+    .notifications(notificationConfig, notificationHandler)
+    .build()
+
+```
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+boolean pushNotificationEnabled = true;
+List<PushDeviceGenerator> pushDeviceGeneratorList = new ArrayList<>();
+NotificationConfig notificationConfig = new NotificationConfig(pushNotificationEnabled, pushDeviceGeneratorList);
+
+NotificationHandler notificationHandler = NotificationHandlerFactory.createNotificationHandler(context, (messageId, channelType, channelId) -> {
+    // Return the intent you want to be triggered when the notification is clicked
+    Intent intent = new Intent();
+
+    return intent;
+});
+
+new ChatClient.Builder("api-key", context)
+        .notifications(notificationConfig, notificationHandler)
+        .build();
+```
+</TabItem>
+</Tabs>
+
+### Customize Notification Style
+
+The SDK lets you define the theming and behavior of the notification UI that users see after they receive a push notification.
+To do this, implement the `NotificationHandler` interface and show your own notification.
+If you want to dismiss notifications when a user visits a channel or logs off of the app, you will need to implement methods for dismissing notifications on the `NotificationHandler` class.
+
+<Tabs>
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin {6,13,17}
```

Comment:
Can you check if highlighted lines are correct?

Reply:
We seem to have updated the code snippet but didn't update highlighted lines. It is already deployed at production with this error. Fortunately, it is not so important.